deploy/roles/mesos_master/defaults/main.yml

```diff
@@ -4,6 +4,10 @@
 # Number of log files to keep at a time
 zookeeper_maxbackupindex: 20
 
+log_file_locations:
```

Comment:
Also, we need to add the permissions tasks to the mesos_agent role as well, although there it is just `/var/log/mesos` and not `/var/log/zookeeper`.  Call that one `mesos_agent_log_file_locations`